Gruntfile.js

```diff
@@ -195,37 +202,30 @@ module.exports = function( grunt ) {
 	} );
 
 	grunt.registerTask( "lint", [
-		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint:dev" ),
+		runIfNewNode( "newer:eslint:dist" )
```

Comment:
On the other hand, I can always invoke `grunt jsonlint eslint` if I need that... 😄  So maybe that's fine.